docs/structured-streaming-programming-guide.md

```diff
@@ -1814,6 +1814,82 @@ Specifically for built-in HDFS state store provider, users can check the state s
 it is best if cache missing count is minimized that means Spark won't waste too much time on loading checkpointed state.
 User can increase Spark locality waiting configurations to avoid loading state store providers in different executors across batches.
 
+### State Store
+
+State store is a versioned key-value store which provides both read and write operations. In
+structured streaming, we use the state store provider to handle the state store operations crossing
```

Comment:
crossing -> across

Reply:
state store operations -> stateful operations

Reply:
Both were done in 7de4b21. Thanks!